python/sanlock.c

```diff
@@ -516,15 +537,15 @@ py_read_lockspace(PyObject *self __unused, PyObject *args, PyObject *keywds)
         goto exit_fail;
 
     /* fill the dictionary information: lockspace */
-    if ((ls_entry = PyString_FromString(ls.name)) == NULL)
+    if ((ls_entry = PyBytes_FromString(ls.name)) == NULL)
```

Comment:
Replacing string with bytes is also a great simple change, please submit it separately. I think you can include all instances of lockspace name (read_lockspace) and resource name (read_resource).